### PR TITLE
conf-mode: T2938: Add format octet-counted for syslog

### PR DESCRIPTION
**Pull request by sever-sever**
Add octet-counted option for syslog.
RFC5424 
option: RSYSLOG_SyslogProtocol23Format

### CODE REVIEW

**Comment by c-po** [comment]
Commit headline should mention service which is affected, syslog in this case.

**Comment by sever-sever** [comment]
> Commit headline should mention service which is affected, syslog in this case.

Sure, Done.

File: data/templates/syslog/rsyslog.conf.tmpl [hunk 1]
```diff
@@ -22,19 +22,23 @@ $outchannel {{file}},{{files[file]['log-file']}},{{files[file]['max-size']}},{{f
 {% if hosts %}
 ## remote logging
 {% for host in hosts %}
-{% if hosts[host]['proto'] == 'tcp' %}
-{% if hosts[host]['port'] %}
+{%   if hosts[host]['proto'] == 'tcp' %}
```
Comment (kwladyka) [inline data/templates/syslog/rsyslog.conf.tmpl:25]: as I understand this is wrong. The `tcp` is not a condition. It should work also for `UDP`. It took my a long time to figure out why `RSYSLOG_SyslogProtocol23Format` doesn't work, but finally!

Comment (kwladyka) [reply data/templates/syslog/rsyslog.conf.tmpl:25]: Also `port` has to be set, so default can't be used. This is very not intuitive and it is not in the doc.

**Pull request merged**